src/plugins/gauge/components/Gauge.vue

```diff
@@ -504,6 +581,11 @@ export default {
                 ]
             });
         },
+        fontSizeFromChars(charNum, charThreshold, startPerc, reducePerc) {
+            const fs = (charNum <= charThreshold) ? startPerc : (startPerc - ((charNum - charThreshold) * reducePerc));
+
+            return fs.toString() + "%";
```

Comment:
can `fs` ever not be a number? e.g., `NaN`?

Reply:
No, it's always a number.